tests/integration/targets/group_vars.yml

```diff
@@ -0,0 +1,12 @@
+---
+# vcenter spicific vars
```

Comment:
can you add here something like "please edit according to your vcenter configs"

Reply:
done